### ISSUE TITLE
There is no function to fill the transparent background with the selected color

### ISSUE THREAD

**Issue by BlackBulletBrony** [alessandrofrancesconi/gimp-plugin-bimp#237]
There is no function to fill the transparent background with the selected color. There is function "plug-in-**colortoalpha**", but there is no function "plug-in-**alphatocolor**", i.e. opposite. Please add it.

**Comment by hayden-t** [comment]
agreed, when i go from tiff to jpg it is filling the transparency with garble.

**Comment by hayden-t** [comment]
try the flatten option
